src/main/70-operational-database.tf

```diff
@@ -0,0 +1,36 @@
+module "aurora" {
+  source = "terraform-aws-modules/rds-aurora/aws"
+
+  name              = "${var.app_name}-operational-database-${var.env}"
+  engine            = "aurora-postgresql"
+  engine_version    = "14.6"
+  engine_mode       = "serverless"
```

Comment:
Missing the `scaling_configuration` setting, with min and max capacity as variables